tokenize.js

```diff
@@ -27,325 +27,284 @@ module.exports = function tokenize(input) {
+        pos = next;
+        break;
 
       default:
-        n = css.charCodeAt(pos + 1)
+        n = css.charCodeAt(pos + 1);
 
         if (code === SLASH && n === ASTERICK) {
-          next = css.indexOf('*/', pos + 2) + 1
-          if (next === 0) unclosed('comment')
+          next = css.indexOf('*/', pos + 2) + 1;
+          if (next === 0) unclosed('comment');
 
-          content = css.slice(pos, next + 1)
-          lines = content.split('\n')
-          last = lines.length - 1
-
-          if (last > 0) {
-            nextLine = line + last
-            nextOffset = next - lines[last].length
-          } else {
-            nextLine = line
-            nextOffset = offset
-          }
+          content = css.slice(pos, next + 1);
 
           tokens.push([
             'comment',
             content,
-            line,
-            pos - offset,
-            nextLine,
-            next - nextOffset
-          ])
-
-          offset = nextOffset
-          line = nextLine
-          pos = next
+            offset,
+            offset + content.length
+          ]);
+
+          offset += content.length;
+          pos = next;
         } else if (code === SLASH && n === SLASH) {
-          RE_NEW_LINE.lastIndex = pos + 1
-          RE_NEW_LINE.test(css)
+          RE_NEW_LINE.lastIndex = pos + 1;
+          RE_NEW_LINE.test(css);
           if (RE_NEW_LINE.lastIndex === 0) {
-            next = css.length - 1
+            next = css.length - 1;
           } else {
-            next = RE_NEW_LINE.lastIndex - 2
+            next = RE_NEW_LINE.lastIndex - 2;
           }
 
-          content = css.slice(pos, next + 1)
+          content = css.slice(pos, next + 1);
 
           tokens.push([
             'comment',
             content,
-            line,
-            pos - offset,
-            line,
-            next - offset,
+            offset,
+            offset + content.length,
             'inline'
-          ])
+          ]);
 
-          pos = next
+          offset += content.length;
+          pos = next;
         } else {
-          RE_WORD_END.lastIndex = pos + 1
-          RE_WORD_END.test(css)
+          RE_WORD_END.lastIndex = pos + 1;
+          RE_WORD_END.test(css);
           if (RE_WORD_END.lastIndex === 0) {
-            next = css.length - 1
+            next = css.length - 1;
           } else {
-            next = RE_WORD_END.lastIndex - 2
+            next = RE_WORD_END.lastIndex - 2;
           }
 
+          content = css.slice(pos, next + 1);
+
           tokens.push([
             'word',
-            css.slice(pos, next + 1),
-            line,
-            pos - offset,
-            line,
-            next - offset
-          ])
-          pos = next
+            content,
+            offset,
+            offset + content.length
+          ]);
+
+          offset += content.length;
+          pos = next;
         }
 
-        break
+        break;
     }
 
     pos++
   }
 
   return tokens
-}
+}
```

Comment:
You need to run Prettier to fix EOF and `;`

Reply:
I have added .prettierrc file. I have applied rules from package.json, but It changes many other files as well. Are u good with that?

Reply:
Strange. Our Prettier config is already in `package.json`, no need for extra file.

Maybe your plugin doesn't see this type of config? Anyway can you remove config back?